docker-compose.yml

```diff
@@ -17,4 +17,7 @@ services:
     depends_on:
       - postgresql
     environment:
-      - SPRING_DATASOURCE_URL=jdbc:postgresql://postgresql:5432/mariopizza
\ No newline at end of file
+      - SPRING_DATASOURCE_URL=jdbc:postgresql://postgresql:5432/mariopizza
+      - CSVFILE=/importfiles/pizza_ingredienten.csv
```

Comment:
I would rename this setting; CSVFILE is too generic, as I also have to import CSV files for orders. Wouldn't want your logic to accidentally start importing order files. Perhaps something like IMPORT_INGREDIENTS_PATH?

Reply:
Outdated